site/.stylelintrc.json

```diff
@@ -23,6 +23,7 @@
     "public/css/sidebar.css",
     "public/css/server.css",
     "public/css/sequence.css",
+    "public/css/system-update.css",
```

Comment:
Is there a particular reason why this new file is being added to the ignored list?

Reply:
I added it since the way I inject styling into the markdown string returned from the server uses a regex to add classes based on the section title. For example, the section labeled SECURITY gets the `update-SECURITY` class, but the linter doesn't like caps in class names. I tried using \L to convert the matching group to lowercase in the regex, but had no luck.  I could probably do another step in processing after the markdown is added as html onto the page to make these classes lowercase, but I thought it might be too messy for what it's worth. Thoughts?

Reply:
Hmm, interesting issue.  I believe the best option here may be to disable stylelint using inline ignores like so:
```
/* stylelint-disable */
a {}
/* stylelint-enable */
```

Reply:
JS' regex does not support the `\L` flag inline to change the casing. You need to apply it as a step on the captured group itself in the replace. The suggested way to do this given the multiple groups (and only want to lowercase one) is to use a function as the second argument to `.replace`, something like:

```
markdown_data = markdown_data.replace(/<li>(\[\w+:(\w+)\].+)(?=<\/li>)|<li>(\[([^\]]+)\].+)(?=<\/li>)/g, (match, p1, p2, p3, p4) => `<li class="release-item"><span class="update-${`${p2}${p4}`.toLowerCase()}">${p1}${p3}</span>`);
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_a_parameter for docs on it.